extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -719,7 +719,10 @@ export class AmpA4A extends AMP.BaseElement {
 
         const consentState = consentResponse[0];
         const consentString = consentResponse[1];
-        const gdprApplies = consentResponse[2];
+        const consentMetadata = consentResponse[2];
+        const gdprApplies = consentMetadata
```

Comment:
I assume that `getAdUrl` will need all information from `consentMetadata` soon. Do you think we should pass the entire object instead? 
cc @Leedehai to review

Reply:
I think this might be the case in the future, but it will require more changes. I think maybe we should  make those changes once they're needed?

Reply:
Yeah I agree with Mica - let's make the changes once new use cases emerge.